doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -372,6 +372,9 @@ and ``coqtop``, unless stated otherwise:
   is intended to be used as a linter for developments that want to be robust to
   changes in the auto-generated name scheme. The options are provided to
   facilitate tracking down problems.
+  A less aggresive version can be enabled with the :flag:`Mangle Names Light`
+  flag which overrides the behavior of :flag:`Mangle Names`, by keeping the
+  original name and only adding the :opt:`Mangle Names Prefix` to the beginging.
```

Comment:
This seems out of place.  This section describes command line options but IIUC you haven't added an command line option to do so (I have no opinion on whether such an option is necessary).

Reply:
You're correct. I will remove this as there is no plan to add a command line option. This option is really for use in interactive development anyhow.